library/include/vulkan/debug/vulkan_profiles.hpp

```diff
@@ -252,7 +252,7 @@ VPAPI_ATTR VkResult vpGetProfileFormatStructureTypes(const VpProfileProperties *
 #include <stdio.h>
 
 #ifndef VP_DEBUG_MESSAGE_CALLBACK
-#define VP_DEBUG_MESSAGE_CALLBACK(MSG) fprintf(stderr, MSG)
+#define VP_DEBUG_MESSAGE_CALLBACK(MSG) fprintf(stderr, "%s\n", MSG)
```

Comment:
Like here, a single "\" which is for the "\n" char but not printing a "\"

Reply:
Two backslash is needed at the python script to get one backslash in the generated code.